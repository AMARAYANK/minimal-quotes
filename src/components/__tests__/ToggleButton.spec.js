/* eslint-disable no-undef */
import React from 'react';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import { ToggleButton } from '../index';

Enzyme.configure({ adapter: new Adapter() });

describe('Testing ToggleButton component', () => {
  it('renders as expected', () => {
    const wrapper = Enzyme.shallow(
      <ToggleButton>
        Hello
      </ToggleButton>,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
