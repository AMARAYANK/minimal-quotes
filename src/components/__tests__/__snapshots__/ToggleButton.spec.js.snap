// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing ToggleButton component renders as expected 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ToggleButton
    selected={false}
  >
    Hello
  </ToggleButton>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "activeOpacity": 0.2,
      "children": <Component
        style={
          Array [
            undefined,
            false,
            false,
          ]
        }
      >
        Hello
      </Component>,
      "selected": false,
      "style": Array [
        undefined,
        undefined,
        false,
        false,
        undefined,
      ],
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": "Hello",
        "style": Array [
          undefined,
          false,
          false,
        ],
      },
      "ref": null,
      "rendered": "Hello",
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "activeOpacity": 0.2,
        "children": <Component
          style={
            Array [
              undefined,
              false,
              false,
            ]
          }
        >
          Hello
        </Component>,
        "selected": false,
        "style": Array [
          undefined,
          undefined,
          false,
          false,
          undefined,
        ],
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "Hello",
          "style": Array [
            undefined,
            false,
            false,
          ],
        },
        "ref": null,
        "rendered": "Hello",
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
